app/views/items/show.html.erb

```diff
@@ -1,17 +1,27 @@
 <% content_for :sidebar do %>
-  <%= sidebar_button_link 'Return to Listing', items_path %>
   <%= edit_item_button %>
   <%= add_repair_button %>
   <%= add_inspection_button %>
 <% end %>
 
 <h3>
+  <%= link_to @item.item_category,
+                item_category_path(@item.item_category) %> >
+  <%= link_to @item.item_type, @item.item_type %> >
+  <%= @item.name %>
+</h3>
+
+<h3>
   <% if @item.status.present? %>
     <%= item_status_label(@item) %> &nbsp;
   <% end %>
-  <%= @item.name %>
+
+  <% if @item.condition.present? %>
+    <%= item_condition_label(@item) %> &nbsp;
```

Comment:
I don't have the bandwidth to handle CSS. (Not one of my stronger suits anyway.) CSS will need to be it's own effort.